### DISCUSSION TITLE
Busco un equipo para Hackaton

### DISCUSSION THREAD

**Discussion by kerizrh** [Launch-X-Latam/MisionFrontEnd#588] [Team Posts]
Hola, todavía busco equipo para el hackaton, chale no me dejen solo que es el primero en el que voy, alguién agregué porfa. 🥲

**Comment by luislordz** [comment]
Hola yo tambien busco equipo  Enviado desde Correo para Windows De: Kevin RiveraEnviado: jueves, 22 de diciembre de 2022 09:10 p. m.CC: SubscribedAsunto: [Launch-X-Latam/mision-backend] Busco un equipo para Hackaton (#129) Hola, todavía busco equipo para el hackaton, chale no me dejen solo que es el primero en el que voy, alguién agregué porfa. 🥲—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***> Libre de virus.www.avast.com
